### DISCUSSION TITLE
证书块到期了 点击申请按钮  后到期日期不更新

### DISCUSSION THREAD

**Discussion by cktk** [1Panel-dev/1Panel#6285] [Q&A]
### 1Panel 版本

v1.10.14-beta.2

### 问题描述

点击申请按钮  证书到期日期不变
![image](https://github.com/user-attachments/assets/3bff6e47-5718-40ab-92a2-af6c0d89dc59)

### 重现步骤

网站->证书管理 -> 找到某一条证书点申请

**Comment by zhengkunwang223** [comment]
刚才试了一下 没有复现
